Source/santactl/Commands/SNTCommandStatus.m

```diff
@@ -81,21 +82,21 @@ + (void)runWithArguments:(NSArray *)arguments daemonConnection:(SNTXPCConnection
   // Kext status
   __block int64_t cacheCount = -1;
   dispatch_group_enter(group);
-  [[daemonConn remoteObjectProxy] cacheCount:^(int64_t count) {
+  [[self.daemonConn remoteObjectProxy] cacheCount:^(int64_t count) {
```

Comment:
#193 that is now in master is conflicting with this change
* rebase against master 
* apply the local `daemonConn` to `self. daemonConn` change.
```
[[self.daemonConn remoteObjectProxy] cacheCounts:^(uint64_t rootCache, uint64_t nonRootCache) {
...
}];
```